ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-day2-provision-chain.yaml

```diff
@@ -8,12 +8,14 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-aws-custom-vpc
   - ref: ipi-conf-aws
+  - ref: ovn-conf
```

Comment:
we have to explicitly set the OVN networking type?

Reply:
@anuragthehatter suggested adding this step, it's required by `network-mtu-migration-decrease`

Reply:
Updated.